src/main/kotlin/org/sd_jwt/SdJwt.kt

```diff
@@ -505,8 +609,16 @@ inline fun <reified T> verifyPresentation(
 
     val sdClaimsParsed = verifyAndBuildCredential(sdJwtParsed, disclosureMap)
 
-    val format = Json { ignoreUnknownKeys = true }
-    return format.decodeFromString(sdClaimsParsed.toString())
+    // Exclude technical claims
+    val sdClaimsParsedFiltered = JSONObject(sdClaimsParsed.toString()).toMap().filterKeys { key ->
+        key !in setOf(
+            SD_DIGEST_KEY,
+            DIGEST_ALG_KEY,
```

Comment:
Is it desirable to exclude these keys? I am not sure how this library is used so I would like to leave the option open for consumers to use these claims.